libs/webb-ui-components/src/constants/index.ts

```diff
@@ -99,7 +99,7 @@ export const DKG_STATS_KEYS_URL = `${DKG_STATS_URL}/#/keys`;
 export const DKG_STATS_AUTHORITIES_URL = `${DKG_STATS_URL}/#/authorities`;
 export const DKG_STATS_PROPOSALS_URL = `${DKG_STATS_URL}/#/proposals`;
 
-export const TANGLE_RPC_ENDPOINT = 'wss://testnet-rpc.tangle.tools';
+export const TANGLE_RPC_ENDPOINT = 'http://localhost:9944';
```

Comment:
Won't this make the Netlify deployment previews or anything that depends on this constant attempt to connect to localhost instead of the testnet?